### PR TITLE
feat(workflow): Add "Delete Rule" button for Issue alerts

### PR DESCRIPTION
**Pull request by billyvg**
This adds a "Delete Rule" button for Issue alerts

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ import recreateRoute from 'app/utils/recreateRoute';
 import space from 'app/styles/space';
 import withApi from 'app/utils/withApi';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx]:
```suggestion
import Confirm from 'app/components/confirm';

```

File: src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx [hunk 2]
```diff
@@ -33,6 +34,7 @@ import recreateRoute from 'app/utils/recreateRoute';
...
 
+import Confirm from 'app/components/confirm';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx]:
```suggestion
```

**Review by matejminar — APPROVED** [review]
Looks good to me.

**Comment by billyvg** [comment]
image
Not sure how that happened :(

**Pull request merged**